src/storage/src/memtable/inserter.rs

```diff
@@ -38,20 +36,20 @@ impl Inserter {
     }
 
     // TODO(yingwen): Can we take the WriteBatch?
```

Comment:
Do you have the answer?

Reply:
I'm not sure but this might be unnecessary to leave a todo here.